src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+
+function ArticleList() {
+  const [articles, setArticles] = useState([]);
+  
+  useEffect(() => {
+    getArticles().then((articles) => {
+      setArticles(articles);
+    });
+  }, []);
+
+  return (
+    <div>
+      {articles.map((article) => {
+        return <ArticleCard key={article.article_id} article={article} />;
+      })}
+    </div>
```

Comment:
Could you add some loading state so we can get some feedback that something happening, on a slow connection?